suite-native/module-development/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@suite-native/module-development",
```

Comment:
Actually IMO, those package names sound strange. `development-module`, `settings-module`, etc would be much clearer 🙏

Reply:
Prefix is better because you have all module packages grouped together in file viewer. We already started to doing that way so I would prefer not rename it unless there will be some strong benefit from that.

Reply:
Yep, makes sense. Unless we create another folder for modules :D Honestly, I am using the file explorer less and less because of the sheer amount of packages anyways, but I suppose it's normal for a monorepo of this scale and I don't even know if using folders to structure would help much...

Reply:
@dahaca Sad to hear that but I can't probably do nothing about it. We need to create small packages because it's way how to avoid cyclic dependencies and also it makes monorepo faster. Also there is like widely accepted idea that flat structure is better than deep nested folder structure, which is something that monorepo offers as side-effect and flat structure is also kind of inevitable in monorepo.

For example here in this PR we run into cyclic dependecies issue, becauase persist related things were placed in in `@suite-native/store` package. Because this package imports reducers from nearly all other modules we were not able to place it to `module-settings`.

We needed to import `storePersistor` back to settings package because there is dev function for storage purge, but it was impossible because it would create cyclic deps (store => module-settings => store). So @bouchja1 can only placed it to dashboard because it was only module without reducer. 

In the end it forced us to create standalone package `@suite-native/storage` where persistor is injected by context. This may sound little bit more complicated but in the it's much much smarted and better because package is now completely standalone with no dependency to any other our code, so we can use it anywhere.

Thanks to that it could be literally taken and used in desktop suite (only if we want to sometimes in the future) or maybe connect-ui with only simple change where you will inject only different storage provider like local storage or smthing.

...

This fuckups with cyclic dependencies are actually feature of monorepo, that guards you agains bad structuring your code and force you to think much more how and where to place things and also more isolate them, which is also very good.